renovate.json

```diff
@@ -68,7 +68,7 @@
     },
     {
       "matchPackageNames": [
-        "quay.io/centos-bootc/builder"
+        "quay.io/centos-bootc/bootc-image-builder"
```

Comment:
Is the bib image specified in this repo as a dependency? If not then this change has no impact. Maybe the section should be removed all together.